fast64_internal/oot/actor/properties.py

```diff
@@ -23,6 +30,52 @@
 ]
 
 
+def getObjName(actorKey: str, paramType: str, paramSubType: str, paramIndex: int):
+    flagTypeToObjName = {"Chest": "chestFlag", "Collectible": "collectibleFlag", "Switch": "switchFlag"}
+    paramTypeToObjName = {
+        "Type": "type",
+        "Property": "props",
+        "Bool": "bool",
+        "Enum": "enum",
+        "ChestContent": "chestContent",
+        "Collectible": "collectibleDrop",
+        "Message": "naviMsg",
+    }
+    suffix = paramTypeToObjName[paramType] if paramType != "Flag" else flagTypeToObjName[paramSubType]
+    return f"{actorKey}.{suffix}{paramIndex}"  # i.e: ``en_test.props1``
```

Comment:
```suggestion
    return f"{actorKey}.{suffix}{paramIndex}"  # e.g.: ``en_test.props1``
```

i.e. is for details https://fr.wiktionary.org/wiki/i.e.
e.g. is for examples https://fr.wiktionary.org/wiki/e._g.#fr